.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Enhancement Request
+description: Is something critical missing? Suggest an enhancement.
+labels:
+  - [enhancement, needs-review]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Before filing an enhancement, please search the existing issues and use the
+        [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
+        feature to add up-votes to existing requests.
+
+
+        When filing an enhancement, please include the following information.
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: >-
+        This project has a [Code of Conduct](https://github.com/vmware/terraform-provider-nsxt/blob/main/CODE_OF_CONDUCT.md)
+        that all participants are expected to understand and follow.
+      options:
+        - label: I have read and agree to the project's Code of Conduct.
+          required: true
+        - label: Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
```

Comment:
Same comment as for bug.YML

Reply:
Resolved in https://github.com/vmware/terraform-provider-nsxt/commit/50fcb7ad1539140b4f6f4a14864b57ee36196e59.